Gordon360/Services/ProfileService.cs

```diff
@@ -79,24 +79,49 @@ public IEnumerable<AdvisorViewModel> GetAdvisors(string id)
             // Stored procedure returns row containing advisor1 ID, advisor2 ID, advisor3 ID 
             var idResult = RawSqlQuery<ADVISOR_SEPARATE_Result>.query("ADVISOR_SEPARATE @ID", idParam).FirstOrDefault();
 
+            //Obeject Array to store advisors information
+            AdvisorViewModel[] advisorsID = new AdvisorViewModel[3];
+
+            // If idResult equal null, it means this user do not have advisor
             if (idResult == null)
             {
-                throw new ResourceNotFoundException() { ExceptionMessage = "No advisor for this id" };
+                //return 0 array advisor list
+                AdvisorIDEnumerable zeroAdvisor = new AdvisorIDEnumerable(advisorsID);
+                List<AdvisorViewModel> zero = new List<AdvisorViewModel>();
+
+                foreach (AdvisorViewModel p in zeroAdvisor)
+                {
+                    if (p != null)
+                    {
+                        zero.Add(p);
+                    }
+                }
+                return zero;
```

Comment:
Can you just replace the whole "if" body with 
    return new List<AdvisorViewModel>();  // empty list because no advisors